File: plugins/textmate/tests/org/jetbrains/plugins/textmate/language/syntax/lexer/PerlLexerTest.java
package org.jetbrains.plugins.textmate.language.syntax.lexer;

import org.jetbrains.plugins.textmate.TestUtil;

public class PerlLexerTest extends LexerTestCase {

  @Override
  protected String getTestDirRelativePath() {
    return "perl";
  }

  @Override
  protected String getBundleName() {
    return TestUtil.PERL;
  }
}

File: plugins/textmate/tests/org/jetbrains/plugins/textmate/language/syntax/lexer/PerlLexerTest.kt
package org.jetbrains.plugins.textmate.language.syntax.lexer

import org.jetbrains.plugins.textmate.TestUtil
import org.jetbrains.plugins.textmate.language.syntax.TextMateLexerTestCase
import org.junit.jupiter.api.Test

class PerlLexerTest : TextMateLexerTestCase() {
  @Test
  fun perl() = doTest("perl.pl", "perl_after.pl")

  @Test
  fun regex() = doTest("regex.pl", "regex_after.pl")

  override val testDirRelativePath = "perl"
  override val bundleName = TestUtil.PERL
}
